docs/installationguide/easydocker.rst

```diff
@@ -0,0 +1,11 @@
+An Easy To Use Docker Image
+===========================
+If you are interested in trying out ZoneMinder quickly, user Dan Landon maintains an easy to use docker image for ZoneMinder. With a few simple configuration changes, it also provides complete Event Server and Machine Learning hook support. Please follow instructions in his repostory. He maintains two repositories:
+
+* If you want to run the latest stable release, please use his `zoneminder repository <https://github.com/dlandon/zoneminder>`__.
+* If you want to run the latest zoneminder master, please use his `zoneminder master repository <https://github.com/dlandon/zoneminder.master-docker>`__.
+
+In both cases, instructions are provided in the repo README files.
+
+If you are looking at making your own docker build of ZoneMinder instead, please refer to :doc:`packpack`.
```

Comment:
packpack isn't really making a docker build of zoneminder... it is about using docker to build binary packages of ZoneMinder to install directly, without docker.  If people want to build their own docker build of zoneminder... they should start with dlandon's Dockerfile.